infrastructure/ssz/src/main/java/tech/pegasys/teku/infrastructure/ssz/SszProfile.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Consensys Software Inc., 2024
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.infrastructure.ssz;
+
+public interface SszProfile extends SszStableContainer {}
```

Comment:
In this approach I wouldn't say that the `SszProfile` IS semantically an `SszStableContainer`. In my mind `SszProfile` is rather an `SszContainer` _incapsulating_ `SszStableContainer`. What do you think?

Reply:
Yes this is actually a good point. I made this decision very early but the actual implementation I ended up following is via incapsulation.  I'll go for extending `SszContainer` unless I find something bad down the line.